SUPPORT_POLICY.md

```diff
@@ -0,0 +1,25 @@
+Distroless is designed to follow debian on their [standard support timeline](https://wiki.debian.org/DebianReleases#Production_Releases).
+
+The current estimation of EOL of images
+
+| Image       |  Support    | Debian 10 EOL | Debian 11 EOL   |
+| ----------- | ----------- | ------------- | --------------- |
+| static      | standard    | aug 2022      | debian 12 + 1yr |
+| base        | standard    | aug 2022      | debian 12 + 1yr |
+| cc          | standard    | aug 2022      | debian 12 + 1yr |
+| java        | standard    | jan 2022      | debian 12 + 3mo |
+| node        | standard    | jan 2022      | debian 12 + 3mo |
+| python      | standard    | jan 2022      | debian 12 + 3mo |
+
+
+Java
+Java will only support current LTS version distributed by debian [see here](https://wiki.debian.org/Java).
```

Comment:
I don't think this works very well for Java (unless you mean all the LTS versions debian is shipping).

There's currently 3 actively supported LTS lines by the main vendors (8, 11, 17).

I assume Debian fixes the LTS version of default-jdk for the lifecycle of the OS version, given it's nearly 3 months since release and its pointing to JDK 11.

 Even so I'm not sure that debian is the best vendor for Java, Java 17 release was very slow, not sure how good they are for others. Would it not be better to get them from a vendor that specialises in Java? e.g. something like Eclipse Temurin or Zulu (which are two of the recommended ones in [actions/setup-java](https://github.com/actions/setup-java#basic). People generally don't want to touch Oracle because of the licensing stuff they've done and reputation.

(this may be off topic from the support policy, fine if so)

Reply:
I think `java-base` should provide users with the sufficient libraries for them to bring their own jre into a container image (either from a provider like temurin/zulu or using something like jlink to create a minimal jre for the application).

Reply:
yeah makes sense, upstreams are pushing `jlink` currently as you can massively decrease image size with it.